src/crimson/os/seastore/async_cleaner.h

```diff
@@ -561,7 +567,7 @@ class JournalTrimmerImpl : public JournalTrimmer {
 
   std::size_t get_trim_size_per_cycle() const final {
     return config.rewrite_backref_bytes_per_cycle +
-      config.rewrite_dirty_bytes_per_cycle;
+      get_dirty_bytes_to_trim();
```

Comment:
Looks to me it is possible for EPM to see the actual `journal_dirty_bytes < config.min_journal_bytes`, right ?

Reply:
I modified `JournalTrimmer::get_alloc_tail_target()` to consider `config.min_journal_bytes`, so EPM wouldn't see actual `journal_dirty_bytes < config.min_journal_bytes`.

I also renamed `rewrite_backref_bytes_per_cycle` to `max_backref_bytes_per_cycle`, because `trim_alloc` doesn't rewrite any thing and the configuration only indicates the maximum number of bytes of new backref extents that `trim_alloc` can generate.

Reply:
See https://github.com/ceph/ceph/pull/63515#discussion_r2134924113